t3/utils/flux.py

```diff
@@ -317,35 +327,47 @@ def run_jsr(gas: ct.Solution,
         V (float, optional): The reactor volume in cm^3.
         a_tol (float, optional): The absolute tolerance for the simulation.
         r_tol (float, optional): The relative tolerance for the simulation.
+        tau_tolerance (float, optional): Cantera will solve JSR equations with step() till tau*tolerance time, after that, it will solve them by advance(t_i). This will adress stiff equestion in which small time steps are required.
```

Comment:
Can you break long lines? See what we did for `reactor_type` in the docstring of `get_profiles_from_simulation`